### ISSUE TITLE
What are the hyperparameters for DeiT-III (epoch 400 or 600)?

### ISSUE THREAD

**Issue by GoJunHyeong** [facebookresearch/deit#214]
Hello.
I would like to re-implement table 15 of the DeiT 3(https://arxiv.org/pdf/2204.07118.pdf).

If you look at the log results for the training hyperparameters you mentioned in Readme, 71.15(paper 79.9) and 74.446(paper 80.4) are recorded in 300 and 400 epochs, respectively.

300 epochs in Table 15 recorded the same 79.7 as the existing ViT.

However, 800 epochs in Table 15 record 81.37 (81.4) as mentioned in readme.

Are the training hyperparameters of 800 epochs and 300 and 400 epochs different in Table 15?
And in order to re-implement the 300, 400 epochs results in Table 15, do we simply need to use the training hyperparameters in DeiT-1 (+increase weight decay from 0.02 to 0.05 and increase stochastic depth drop-rate by 0.05 every 200 epochs)?
